aiida/manage/configuration/profile.py

```diff
@@ -130,12 +131,18 @@ def __init__(self, name, attributes, from_config=False):
 
     def get_repository(self) -> 'Repository':
         """Return the repository configured for this profile."""
-        from disk_objectstore import Container
         from aiida.repository import Repository
+
+        backend = self.get_backend()
+        return Repository(backend=backend)
+
+    def get_backend(self) -> 'DiskObjectStoreRepositoryBackend':
```

Comment:
I wouldn't call this `get_backend`. It would be too confusing with the database backend. If we really need this method, a better name would be `get_repository_backend`. However, why can't we just call `get_repository().backend`? Is it necessary to have this intermediate method?